src/components/MainPage.js

```diff
@@ -0,0 +1,188 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Button from "@mui/material/Button";
+import CameraIcon from "@mui/icons-material/PhotoCamera";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Labels from "./Labels";
+import { Chip } from "@mui/material";
+import Progress from "./Progress";
+import reactJsLogo from "../logoImages/react-logo.png";
```

Comment:
I usually create a folder `assets` and inside of it I create a folder `images`. I've seen this pattern before too